### ISSUE TITLE
toast close missing focus indicator

### ISSUE THREAD

**Issue by thiessenp-cds** [cds-snc/platform-forms-client#7918]
When a toast is displayed the close button should show a focus indicator when focussed.

This issue was found during an accessibility evaluation. (attached)

[150 - Styling element outlines and borders in a way that removes or renders non-visible the visual focus indicator - 2 (TBD so just verify fixed).pdf](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeTcvRGc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--4dfbde63aa95f9dee86925da6579941141ed7915/150%20-%20Styling%20element%20outlines%20and%20borders%20in%20a%20way%20that%20removes%20or%20renders%20non-visible%20the%20visual%20focus%20indicator%20-%202%20%28TBD%20so%20just%20verify%20fixed%29.pdf)
